Containerfile.common

```diff
@@ -61,6 +61,17 @@ RUN --mount=type=cache,dst=/var/cache/dnf \
     done && \
     find /var/cache/rpms
 
+# Remove kernel version from kmod package names
+# FIXME: This will only work for 6.* kernels unfortunately
+# FIXME: The sed is a gross hack, maybe PR upstream?
```

Comment:
I agree the sed for adding `--buildroot` to `fpm` is a good upstream PR candidate as it is adding the same/similar behavior which was already added to `rpmrebuild` as of the newer version in Fedora 41.

Reply:
Oh i see the PR here, nice! https://github.com/jordansissel/fpm/issues/2076